pom.xml

```diff
@@ -15,7 +15,8 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>14</version>
+    <version>18</version>
+    <relativePath/>
```

Comment:
Do we need this element present in pom.xml?

Reply:
Yes, to make maven happy. Otherwise, it expects to find parent pom in a parent directory and if finds one (for example if there is an uber pom that builds multiple projects), maven will complain.

Reply:
But it was happy before :) plus I don't see this element present in pom.xml of other Apache projects (Calcite, Camel). Does Drill structure differ?

Reply:
Try to add an empty pom to the parent of your drill repo :). I guess most of the projects assume that there will be no pom.xml file in a parent directory and it is not always the case, so it is better to follow maven guidelines (http://maven.apache.org/ref/3.0/maven-model/maven.html)

Reply:
I have tried, it does fail ) Thanks for explanation.